demo/streaming_filter_unified/run.py

```diff
@@ -0,0 +1,98 @@
+import gradio as gr
```

Comment:
Nit: I think the "flip" is not the most compelling initial state for a demo showing the unified input/output image streaming. "edges" makes a bit more sense. Would even cooler if we could add a real object detection demo with YolovX

Reply:
Yes I think the current state is `flip`. A demo with yolo would be cool as well!